### ISSUE TITLE
Automatically install and enable a custom Steam Controller configuration for gamepad

### ISSUE THREAD

**Issue by Dylan-Perry** [Wildlander-mod/Launcher#626]
We've got fancy new gamepad controls that would be ideal to ship and enable by default on the user's machine.

This involves three tasks:

1. Installing the custom config file to the proper location. Directory is Steam\userdata\\<user's ID number>\\241100\remote\controller_config\489830

![steamcontroller](https://user-images.githubusercontent.com/48102456/182409803-b4baddef-39d7-42df-981e-a8b89d2d856e.jpg)

2. Updating the user's steam configuration settings to enable steam controller support for Skyrim SE specifically. This is accomplished by backing up, and then replacing the existing `localconfig.vdf` file with my edited version. This should occur within the directory Steam\userdata\\<user's ID number>\\config
 
![localconfig](https://user-images.githubusercontent.com/48102456/182410707-d77e7903-7f02-4a30-865b-1e6d730e3860.png)

3. On game exit, we must restore the user's original `localconfig.vdf` file within the directory specified in task 2. This file is responsible for determining many settings across Steam games, and we must restore it so we don't adversely affect the user's experience in other steam games.

We also technically have to grab the user's Steam ID number, but Matt said this is possible via the command line.

Basically, we are looking at one permanent file delivery that we don't have to worry about once placed (actual controller config), and a second temporary file delivery that should institute on game start, and "revert" on game end (steam general config).

The files will be contained within the modpack itself:

- `launcher/controller/controller-config_wl.vdf`
- `launcher/controller/localconfig.vdf`

**Comment by MattLish** [comment]
It isn't totally clear how this works currently.  Waiting on Dylan to confirm the functionality.
